.github/workflows/dogfood.yaml

```diff
@@ -101,22 +104,16 @@ jobs:
         id: message
         run: echo "pr_title=$(git log --format=%s -n 1 ${{ github.sha }})" >> $GITHUB_OUTPUT
 
-      - name: "Get latest Coder binary from the server"
-        if: github.ref == 'refs/heads/main'
-        run: |
-          curl -fsSL "https://dev.coder.com/bin/coder-linux-amd64" -o "./coder"
-          chmod +x "./coder"
-
       - name: "Push template"
         if: github.ref == 'refs/heads/main'
         run: |
-          ./coder templates push $CODER_TEMPLATE_NAME --directory $CODER_TEMPLATE_DIR --yes --name=$CODER_TEMPLATE_VERSION --message="$CODER_TEMPLATE_MESSAGE"
+          terraform apply -auto-approve
         env:
           # Consumed by Coder CLI
           CODER_URL: https://dev.coder.com
           CODER_SESSION_TOKEN: ${{ secrets.CODER_SESSION_TOKEN }}
           # Template source & details
-          CODER_TEMPLATE_NAME: ${{ secrets.CODER_TEMPLATE_NAME }}
-          CODER_TEMPLATE_VERSION: ${{ steps.vars.outputs.sha_short }}
-          CODER_TEMPLATE_DIR: ./dogfood
-          CODER_TEMPLATE_MESSAGE: ${{ steps.message.outputs.pr_title }}
+          TF_VAR_coder_template_name: ${{ secrets.CODER_TEMPLATE_NAME }}
```

Comment:
Is it case insensitive?

Reply:
Apparently so

Reply:
Yeah, I can't get it to pick up the all-caps version:
```tf
variable "CODER_TEMPLATE_NAME" {
  type = string
}
```
```bash
$ export CODER_TEMPLATE_NAME="coder"
$ echo "$CODER_TEMPLATE_NAME"
coder
```

```
│ Error: Invalid Attribute Value Length
│ 
│   with coderd_template.dogfood,
│   on main.tf line 45, in resource "coderd_template" "dogfood":
│   45:   name            = var.CODER_TEMPLATE_NAME
│ 
│ Attribute versions[0].name string length must be at least 1, got: 0
```

Reply:
Shouldn't it be
```
$ export TF_VAR_CODER_TEMPLATE_NAME="coder"
```
For terraform to see it as an input variable?

Reply:
So you are doing it the correct way. :)

Reply:
Yeah, oops, I typo'd it the second time around. I must've also done something wrong the first time - fixed, thank you :)